backend/config/settings.py

```
@@ -25,17 +25,24 @@ def load_dotenv(dotenv_path: Path) -> None:
-print("DEBUG - REDIS_URL present:", "REDIS_URL" in os.environ)
-if "REDIS_URL" in os.environ:
-    print("DEBUG - REDIS_URL prefix:", os.environ["REDIS_URL"][:15])
-
-SECRET_KEY = os.getenv(
-    "SECRET_KEY", "django-insecure-dev-key-not-for-production-use-32bytes!!"
-)
+# SECURITY: Require SECRET_KEY in production. Only use insecure fallback in development.
+SECRET_KEY = os.getenv("SECRET_KEY")
+if not SECRET_KEY:
+    import warnings
+    if DEBUG:
```

Ruff (0.15.20)

[error] Undefined name `DEBUG`

(F821)